### ISSUE TITLE
isUpdated() returns undefined

### ISSUE THREAD

**Issue by JetJet13** [contentful/contentful-management.js#129]
Hey there,

It appears that the method `isUpdated()` for `entries` and `assets` returns `undefined` when the `entry` or `asset` is in `draft` mode (regardless if the `entry` or `asset` has a previously published version). 

Here's an example,
```javascript
const client = this.managementClient

return client.getSpace(this.offerSpace)
       .then((space) => {
            return space.getEntry(query)
       })
       .then((entry) => {
            entry.isDraft() // === true
            return entry.isUpdated() // === undefined
       })
```
## Culprit
The issue seems to be with the function [createUpdatedChecker()](https://github.com/contentful/contentful-management.js/blob/master/lib/instance-actions.js#L85), which returns the following
```javascript
return this.sys.publishedVersion && this.sys.version > this.sys.publishedVersion + 1
```

## Fix
we can correct the boolean check by adding an explicit check of the first condition, ie)
```javascript
return (this.sys.publishedVersion !== undefined && this.sys.version > this.sys.publishedVersion + 1)
```

I can create a small PR to put in the fix if necessary, because I would like to remove the workaround I currently have in place as soon as possible. 

Thanks.

**Comment by axe312ger** [comment]
Hey @JetJet13,

thank you for your report. I fixed this and published version `v3.10.0`.

Let me know if it is still an issue,
Benedikt

**Comment by JetJet13** [comment]
Hey Benedikt,

`isUpdated()` works as expected now. Thanks for the quick fix.

Cheers.
